src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -68,23 +68,23 @@ class OppgaveService(
         }
     }
 
-    private fun hentForrigeBehandling(iverksett: Iverksett): Iverksett {
+    private fun hentForrigeBehandling(iverksett: Iverksett): Iverksett? {
         return iverksett.behandling.forrigeBehandlingId?.let {
             iverksettingRepository.hent(it)
-        } ?: error("Mangler forrigeBehandlingId på revurdering for behandling=${iverksett.behandling.behandlingId}")
+        } ?: null
     }
 
     private fun aktivitetEllerPeriodeEndret(iverksett: Iverksett): Boolean {
         return harEndretAktivitet(iverksett) || harEndretPeriode(iverksett)
     }
```

Comment:
Hadde man kunnet gjort det sånn her? 
Skal den returnere true/false? 

```suggestion
private fun aktivitetEllerPeriodeEndret(iverksett: Iverksett): Boolean {
        if (iverksett.behandling.forrigeBehandlingId == null) return false
        val forrigeBehandling = hentForrigeBehandling(iverksett)
        return harEndretAktivitet(iverksett, forrigeBehandling) || harEndretPeriode(iverksett, forrigeBehandling)
    }

    private fun harEndretAktivitet(iverksett: Iverksett, forrigeBehandling: Iverksett): Boolean {
        return iverksett.gjeldendeVedtak().aktivitet != forrigeBehandling.gjeldendeVedtak().aktivitet
    }

    private fun harEndretPeriode(iverksett: Iverksett, forrigeBehandling: Iverksett): Boolean {
        return iverksett.vedtaksPeriodeMedMaksTilOgMedDato() != forrigeBehandling.vedtaksPeriodeMedMaksTilOgMedDato()
    }
```

Reply:
Godt poeng! Legger til det